Makefile

```diff
@@ -340,8 +341,13 @@ endif
 		--set webhooks.enabled=${WEBHOOKS_ENABLED} \
 		aiven-operator charts/aiven-operator
 
+.PHONY: goimports
```

Comment:
Add `goimports` to all the make commands where we need it, like build, run, etc

Reply:
It looks like `goimports` command is only used in the `imports` target which is listed a dependency to `generate`, which in turn is the dependency to `build` and `run`. So as far as I understand having `goimports` as a dependency to `imports` target is enough to get the binary downloaded on demand when running `make build` or `make run`. Tested with

```
❯ rm bin/goimports
❯ make build
go generate ./...
2023/12/18 12:41:17 can't compile field "topic_name" regex `^(?!\.$|\.\.$)[-_.A-Za-z0-9]+$`: error parsing regexp: invalid or unsupported Perl syntax: `(?!`
2023/12/18 12:41:17 can't compile field "name" regex `^(?!\.$|\.\.$)[-_.A-Za-z0-9]+$`: error parsing regexp: invalid or unsupported Perl syntax: `(?!`
2023/12/18 12:41:17 can't compile field "tag" regex `^(?!aiven-)[^\W\d_](?:[:\w./-]*[\w./-])?$`: error parsing regexp: invalid or unsupported Perl syntax: `(?!`
2023/12/18 12:41:17 can't compile field "kafka_topic" regex `^(?!\.$|\.\.$)[-_.A-Za-z0-9]+$`: error parsing regexp: invalid or unsupported Perl syntax: `(?!`
test -s /Users/riski/code/work/aiven-operator/bin/controller-gen || GOBIN=/Users/riski/code/work/aiven-operator/bin go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.9.2
/Users/riski/code/work/aiven-operator/bin/controller-gen object:headerFile="hack/boilerplate.go.txt" paths="./..."
test -s /Users/riski/code/work/aiven-operator/bin/goimports || GOBIN=/Users/riski/code/work/aiven-operator/bin go install golang.org/x/tools/cmd/goimports@latest
find . -type f -name '*.go' -exec sed -zi 's/"\n\+\t"/"\n"/g' {} +
/Users/riski/code/work/aiven-operator/bin/goimports -local "github.com/aiven/aiven-operator" -w .
/Users/riski/code/work/aiven-operator/bin/controller-gen rbac:roleName=manager-role crd:allowDangerousTypes=true webhook paths="./..." output:crd:artifacts:config=config/crd/bases
go run ./generators/docs/...
go run ./generators/charts/... --version= --operator-charts ./charts/aiven-operator --crd-charts ./charts/aiven-operator-crds
[ "" == "" ] || sed '/## \[/d' ./CHANGELOG.md > docs/docs/changelog.md
go fmt ./...
go vet ./...
go build -o bin/manager main.go
```
and it downloads the `goimports` to localbin on demand.